sympy/combinatorics/free_groups.py

```diff
@@ -150,6 +150,7 @@ def __new__(cls, symbols):
             obj.dtype = type("FreeGroupElement", (FreeGroupElement,), {"group": obj})
             obj.symbols = symbols
             obj.generators = obj._generators()
+            obj.relators = tuple()
```

Comment:
This could rather be a class attribute, a common property of all free groups instead of an instance attribute in every groups dictionary.